apps/files/tests/test_helpers.py

```diff
@@ -201,7 +201,18 @@ def test_contents_size(self):
         self.assertRaises(forms.ValidationError, self.viewer.extract)
 
     def test_default(self):
-        eq_(self.viewer.get_default(None), 'install.rdf')
+        self.viewer.extract()
+        assert self.viewer.get_default(None) == 'install.rdf'
+
+    def test_default_webextension(self):
+        viewer = FileViewer(make_file(2, get_file('webextension.xpi')))
+        viewer.extract()
+        assert viewer.get_default(None) == 'manifest.json'
+
+    def test_default_package_json(self):
+        viewer = FileViewer(make_file(3, get_file('new-format-0.0.1.xpi')))
+        viewer.extract()
+        assert viewer.get_default(None) == 'package.json'
 
```

Comment:
There is no test for the `else None` part of the return statement in case none of the files were found.